### DISCUSSION TITLE
DBNA profile

### DISCUSSION THREAD

**Discussion by jcarbajalpeppolsotf** [phax/phase4#214] [General]
Hello Phillip, any clue on testing client with DBNA profile?
We have already setup C3, however we are having issues when testing phase4 client with such profile

**Comment by ri4a** [comment]
I am assuming TEST network.

For sending: have you tried sending to GLN::1200109963131?
For receiving: what is your test identifier to send to?

**Comment by phax** [comment]
@jcarbajalpeppolsotf unfortunately I don't have any official information on testing with the DBNA profile. As soon as I know more, I will keep you posted. I am therefore converting this into a discussion
